rakelib/kokoro/kokoro.rb

```diff
@@ -163,6 +163,32 @@ def all_local_docs_tests only_updated: false
     end
   end
 
+  def one_local_cloudrad_docs gem
```

Comment:
This class is (mostly?) dead. The logic was moved to `.toys/kokoro/.lib/releaser.rb` a few months ago.